tests/suites/test_suite_x509write.function

```diff
@@ -298,11 +298,12 @@ void x509_csr_check_opaque(char *key_file, int md_type, int key_usage,
     mbedtls_test_rnd_pseudo_info rnd_info;
 
     mbedtls_x509write_csr_init(&req);
+    mbedtls_pk_init(&key);
```

Comment:
Non-blocker: I prefer to init immediately after the declaration. The reason a lot of code has all the declarations then all the function-call initializations is that it was written for C89, but now we use C99 which allows function calls mixed with declarations.